### PR TITLE
Add participant data export to google sheets

### PR DESCRIPTION
**Pull request by shumatepf**

### CODE REVIEW

File: gdrive/export_client.py [hunk 1]
```diff
@@ -172,3 +170,20 @@ def get_qualtrics_response(surveyId: str, responseId: str):
             f"No survey response found for responseId: {responseId}"
         )
     return r.json()
+
+
+def upload_spreadsheet(first, last, email, responseId, time):
```
Comment (dzaslavskiy) [inline gdrive/export_client.py:175]: perhapps export upload_spreadsheet can be moved to client.py so the two client dont need to refrence each other. Also the function name seems wrong.

File: gdrive/export_client.py [hunk 2]
```diff
@@ -172,3 +170,20 @@ def get_qualtrics_response(surveyId: str, responseId: str):
...
+    values = [[first, last, first + " " + last, email, responseId, time]]
+
+    body = {"values": values}
+    result = (
```
Comment (dzaslavskiy) [inline gdrive/export_client.py:179]: should check if result is success

**Review by dzaslavskiy — CHANGES_REQUESTED** [review]

File: gdrive/export_client.py (hunk 1 above)
Comment (shumatepf) [reply gdrive/export_client.py]: Good idea, moved and name changed

**Review by dzaslavskiy — DISMISSED** [review]

File: gdrive/export_client.py (hunk 2 above)
Comment (shumatepf) [reply gdrive/export_client.py]: Added check

File: gdrive/client.py [hunk 3]
```diff
@@ -164,3 +164,26 @@ def delete_file(id: str) -> None:
...
+
+
+def upload_participant(first, last, email, responseId, time):
+    """
+    Append participant data to spreadsheet
+    """
+    values = [[first, last, first + " " + last, email, responseId, time]]
+
+    body = {"values": values}
+    result = (
```
Comment (dzaslavskiy) [inline gdrive/client.py:176]: will result be none if append fails?

**Review by dzaslavskiy — APPROVED** [review]

**Review by dzaslavskiy — APPROVED** [review]

**Pull request merged**